### PR TITLE
Added missing rpc urls and chain ids

### PR DESCRIPTION
**Pull request by VGabriel45**
# Summary

Added SEPOLIA and ARBITRUM_SEPOLIA rpc & enum as default for cases where rpc url is not provided.

Related Issue: #DEVX-414

## Change Type

- [x] Bug Fix
- [ ] Refactor
- [x] New Feature
- [ ] Breaking Change
- [ ] Documentation Update
- [ ] Performance Improvement
- [ ] Other

# Checklist

- [x] My code follows this project's style guidelines
- [x] I've reviewed my own code
- [ ] I've added comments for any hard-to-understand areas
- [ ] I've updated the documentation if necessary
- [x] My changes generate no new warnings
- [ ] I've added tests that prove my fix is effective or my feature works
- [x] All unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published

### CODE REVIEW

**Review by livingrockrises — APPROVED** [review]
approving. make sure yarn build and yarn test works

File: packages/core-types/src/ChainsTypes.ts [hunk 1]
```diff
@@ -28,5 +30,10 @@ export enum ChainId {
   ASTAR_TESTNET = 81,
   CHILIZ_MAINNET = 88888,
   CHILIZ_TESTNET = 88882,
+  CORE_MAINNET = 1116,
```
Comment (livingrockrises) [inline packages/core-types/src/ChainsTypes.ts:33]: Are these already added in ChainId in core-types?

**Review by AmanRaj1608 — APPROVED** [review]
Should also update the `bundler/Constants.ts`. 

Doesn't need this as it can use default values, but just for consistency
- UserOpReceiptIntervals
- UserOpWaitForTxHashIntervals
- UserOpReceiptMaxDurationIntervals
- UserOpWaitForTxHashMaxDurationIntervals

File: packages/core-types/src/ChainsTypes.ts (hunk 1 above)
Comment (VGabriel45) [reply packages/core-types/src/ChainsTypes.ts:33]: This is the chainId enum in core-types package

**Comment by VGabriel45** [comment]
> Should also update the `bundler/Constants.ts`.
> 
> Doesn't need this as it can use default values, but just for consistency
> 
> * UserOpReceiptIntervals
> * UserOpWaitForTxHashIntervals
> * UserOpReceiptMaxDurationIntervals
> * UserOpWaitForTxHashMaxDurationIntervals

Added

**Pull request merged**